cookbook/tools/twilio_tools.py

```diff
@@ -0,0 +1,38 @@
+from phi.agent import Agent
+from phi.tools.twilio import TwilioTools
+from phi.model.openai import OpenAIChat
+
+"""
+Example showing how to use the Twilio Tools with Phi.
+
+Requirements:
+- Twilio Account SID and Auth Token (get from console.twilio.com)
+- A Twilio phone number
+- pip install twilio
+
+Usage:
+- Set the following environment variables:
+    export TWILIO_ACCOUNT_SID="your_account_sid"
+    export TWILIO_AUTH_TOKEN="your_auth_token"
+
+- Or provide them when creating the TwilioTools instance
+"""
+
+
+agent = Agent(
+    name="Twilio Agent",
+    instructions=[
+        """You can help users by:
+        - Sending SMS messages
+        - Checking message history
+        - getting call details
+        
+        Always confirm before sending messages."""
```

Comment:
Hey @jayeshp19  can you remove this line from instructions as in this example we can confirm.
`Always confirm before sending messages`

Reply:
Hi @anuragts Thanks for pointing out. I've updated code.